### PR TITLE
update cloundant to 3.x and other packages

### PR DESCRIPTION
**Pull request by csantanapr**
nodejs 8:
  - update marked  from `0.5.1` to `0.5.2`
  - update mongodb from `3.1.9` to `3.1.10`
  - update nodemailer from `4.6.8` to `4.7.0`
  - update superagent from `3.8.3` to `4.0.0`
  - update twilio from `3.23.2` to `3.24.0`
  - update ws from `6.1.0` to `6.1.2`

nodejs10
  - update @cloudant/cloudant  from `2.4.1` to `3.0.0`
  - update marked  from `0.5.1` to `0.5.2`
  - update mongodb from `3.1.9` to `3.1.10`
  - update nodemailer from `4.6.8` to `4.7.0`
  - update superagent from `4.0.0-beta.5` to `4.0.0`
  - update twilio from `3.23.2` to `3.24.0`
  - update ws from `6.1.0` to `6.1.2`

### CODE REVIEW

File: nodejs10/CHANGELOG.md [hunk 1]
```diff
@@ -2,11 +2,90 @@
 
 ## Migrating from `nodejs:8` to `nodejs:10`
 - The `ibm_db` npm package is not available in `nodejs:10`. The `ibm_db` currently doesn't support nodejs10, track progress in [issue ibmdb/node-ibm_db/issues/482](https://github.com/ibmdb/node-ibm_db/issues/482#issuecomment-436895541) expected on Dec. 7th 2018
-- The `cloudant` npm package is not available in `nodejs:10`, the package is deprecated, you need to use the official npm package [@cloudant/cloudant](https://www.npmjs.com/package/@cloudant/cloudant) when importing the nodejs module (i.e `require('@cloudant/cloudant')).
+- The `cloudant` npm package is not available in `nodejs:10`, the package is deprecated, you need to use the official npm package [@cloudant/cloudant](https://www.npmjs.com/package/@cloudant/cloudant)v3.0.0 when importing the nodejs module (i.e `require('@cloudant/cloudant')) also the [v3.x only return Promises](https://github.com/cloudant/nodejs-cloudant/blob/master/api-migration.md#2x--3x).
```
Comment (mdeuser) [inline nodejs10/CHANGELOG.md]: add a space before `v3.0.0`

Comment (mdeuser) [inline nodejs10/CHANGELOG.md]: are the quotes correct here:   `require('@cloudant/cloudant'))

Comment (csantanapr) [reply nodejs10/CHANGELOG.md]: is missing back tick, I added it

Comment (csantanapr) [reply nodejs10/CHANGELOG.md]: add the space

Comment (mdeuser) [inline nodejs10/CHANGELOG.md]: rewording   `also the [v3.x only return Promises]`
-> `also [v3.x only returns Promises]`

Comment (csantanapr) [reply nodejs10/CHANGELOG.md]: updated

**Pull request merged**